code/modules/projectiles/projectile/bullets.dm

```diff
@@ -192,22 +192,22 @@
 
 /obj/item/projectile/bullet/rifle
 	fire_sound = 'sound/weapons/gunshot/gunshot3.ogg'
-	damage = 45
+	damage = 40
 	armor_penetration = 25
 	penetration_modifier = 1.5
 	penetrating = 1
 	distance_falloff = 1.5
 
 /obj/item/projectile/bullet/rifle/military
 	fire_sound = 'sound/weapons/gunshot/gunshot2.ogg'
-	damage = 40
+	damage = 35
 	armor_penetration = 35
 	penetration_modifier = 1
 
 /obj/item/projectile/bullet/rifle/t12
 	fire_sound = "gun_t12"
-	damage = 24
-	armor_penetration = 40
+	damage = 25
+	armor_penetration = 50
```

Comment:
I like the idea of a super-penetration weapon.
Can you decrease damage but increase penetration even more?

Reply:
Anything less than 25 damage and it becomes weaker than most **melee** weapons, which is... yeah, not good.

Reply:
I'll give it a go then. It's not like T12 is normally spawned/used outside of admemery, at the moment.